diff.go

```diff
@@ -850,22 +849,22 @@ func DiffBlobs(oldBlob *Blob, oldAsPath string, newBlob *Blob, newAsPath string,
 	return nil
 }
 
-type ApplyHunkCallback func(*DiffHunk) (apply bool, err error)
-type ApplyDeltaCallback func(*DiffDelta) (apply bool, err error)
+type ApplyHunkCb func(*DiffHunk) (apply bool, err error)
+type ApplyDeltaCb func(*DiffDelta) (apply bool, err error)
```

Comment:
```suggestion
type ApplyHunkCallback func(*DiffHunk) (apply bool, err error)
type ApplyDeltaCallback func(*DiffDelta) (apply bool, err error)
```

the older name was better for consistency:

```shell
~/git2go$ git grep 'type.*Cb func' | wc -l
0
~/git2go$ git grep 'type.*Callback func' | wc -l
24
```

in L860-L861, the same name can be used for the field and the type, which is the way to express that something is not to be embedded.

Reply:
Fixed